### PR TITLE
Update logo

### PR DESCRIPTION
**Pull request by esadek**
Removed old logo and added better resolution new logo.

**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://rudderstack.com/">
-    <img src="https://user-images.githubusercontent.com/59817155/121357083-1c571300-c94f-11eb-8cc7-ce6df13855c9.png">
```
Comment (SumanthPuram) [inline README.md:3]: I use Github's dark theme. It is difficult to see the logo in that.
Attaching the older one and the new one for reference

Screenshot 2022-01-06 at 9 14 02 AM

Screenshot 2022-01-06 at 9 13 45 AM

**Review by SumanthPuram — COMMENTED** [review]
Should we add a background in this?

**Comment by arnab-p** [comment]
We should update the Architecture diagram as well.
Here is my recommendation
rs-arch-1

**Comment by esadek** [comment]
@arnab-p Thoughts?

architecture

File: README.md (hunk 1 above)
Comment (esadek) [reply README.md:3]: Done

**Review by SumanthPuram — APPROVED** [review]

**Comment by SumanthPuram** [comment]
> We should update the Architecture diagram as well. Here is my recommendation rs-arch-1

How is this different from the existing diagram?

**Review by ameypv-rudder — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
We can use @esadek 's version as the flow looks cleaner, the destinations have categories instead of individual tools.

**Comment by SumanthPuram** [comment]
@ameypv-rudder @esadek Let's merge this?

**Pull request merged**